api/FederatedCredential.json

```
@@ -24,7 +25,11 @@
           "opera": "mirror",
           "opera_android": "mirror",
           "safari": {
-            "version_added": false
+            "version_added": false,
+            "impl_url": [
+              "https://webkit.org/b/167375",
+              "https://webkit.org/b/172014"
+            ]
```

The specific bug should be sufficient:

```suggestion
            "impl_url": "https://webkit.org/b/172014"
```